src/main/java/uk/gov/hmcts/divorce/divorcecase/constant/ConditionalOrderCourtConstants.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.divorce.divorcecase.constant;
+
+public final class ConditionalOrderCourtConstants {
+
+    public static final String birmingham = "Birmingham Civil and Family Justice Centre";
+    public static final String buryStEdmunds = "Bury St. Edmunds Regional Divorce Centre";
```

Comment:
not sure we need this but should be fine

Reply:
Java does not like constants in enums